hooks/common/mutations.ts

```diff
@@ -19,19 +20,19 @@ type CommentParamProps<T> = T extends "post"
       };
 
 interface CommentRequestProps {
-  id: number;
+  id: string;
   comment?: string;
   commentId?: string;
 }
-export const useGroupCommentMutation = <T extends "post" | "patch" | "delete">(
+
+export const useCommentMutation = <T extends "post" | "patch" | "delete">(
   method: T,
   type: "gather" | "group",
-  id: number,
+  id: string,
   options?: MutationOptions<CommentParamProps<T>>,
 ) =>
   useMutation<void, AxiosError, CommentParamProps<T>>((param) => {
-    const url = type === "group" ? "groupStudy" : type + "/comment";
-
+    const url = (type === "group" ? "groupStudy" : type) + "/comment";
```

Comment:
`type` 값이 `"gather"`, `"group"` 말고도 많아질 예정이니까, 객체 맵을 만들어서 url을 관리해도 괜찮아보여요

```suggestion
    const commentApiUrlMap = {
      group: "groupStudy",
      gather: "gather"
      // ... mapping other type and url
    }
    const url = `commentApiUrlMap[type]/comment`;
```